entente/test_rigid_transform.py

```diff
@@ -89,6 +89,17 @@ def test_rigid_rotation_single_point():
 
 
 @pytest.mark.skipif(env_flag("CI") is True, reason="failing in CI when numpy>=1.19.3")
+# TODO: Clarify the intent of this test.
+#
+# This test produces different answers on numpy 1.19.3 and higher on certain architectures. This is
+# because there are duplicate singular values.
+#
+# Since a reflection through the origin in 3-space is not a rotation at all, and there is no rotation matrix
+# which matches the effect of the reflection through origin, it's not clear what the correct result
+# should be.
```

Comment:
```suggestion
# Since a reflection through the origin in 3-space is not a rotation at all, and there is no rotation
# matrix which can reflect through the origin, it's not clear what the correct result should be.
```